azure-pipelines.yml

```diff
@@ -96,7 +94,7 @@ jobs:
       displayName: Install dependencies
     - script: |
         cd hypothesis-python
-        pytest
+        pytest tests\cover  tests/datetime tests\numpy  tests\pandas tests\py3  tests\pytest
```

Comment:
I'm a little nervous about leaving `nocover` and `quality` untested on Windows, and would like to keep testing the other extras.  Perhaps we could split them into a separate job so they can run concurrently?

Reply:
`quality` in particular really aren't worth running multiple times - they're things that are intrinsically just testing our data distribution and are almost entirely platform independent. My feeling was that `nocover` is broadly the same.

I don't think running concurrently would be especially helpful - we spend a lot of time waiting on startup if we do that.